packages/schemas/src/seeds/cloud-api.ts

```diff
@@ -17,6 +17,8 @@ export enum CloudScope {
   ManageTenantSelf = 'manage:tenant:self',
   SendSms = 'send:sms',
   SendEmail = 'send:email',
+  /** The user can access external (independent from Logto instance) resource to run JWT payload customizer scripts. */
+  CustomJwt = 'custom:jwt',
```

Comment:
'custom:jwt' does not sound appropriate.   Instead of customizing a jwt, the scope is more like executing an action?

Reply:
Yes, it's the access to executing an action. And the naming aligns with other service scopes (`send:sms` and `send:email`).

Reply:
Discussed offline, temporarily name cloud scope in verb+noun convention and let the name of the scope match its business purpose.

Reply:
Updated to be `fetch:custom:jwt`.